process_pr.py

```diff
@@ -740,6 +738,13 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
     # For future pre_checks
     # if prId>=somePRNumber: default_pre_checks+=["some","new","checks"]
     pre_checks_url = {}
+    events = {}
+    commit_cache = {}
+    all_commits = []
+    ok_too_many_commits = False
+    warned_too_many_commits = False
+
+    l2s = ", ".join([gh_user_char + name for name in CMSSW_ISSUES_TRACKERS])
```

Comment:
I think this is not needed, you can move `", ".join([gh_user_char + name for name in CMSSW_ISSUES_TRACKERS])` to directly the place where we create the comment

Reply:
Done.